client/app/core/catalogs-state.service.js

```diff
@@ -97,7 +97,7 @@ export function CatalogsStateFactory(CollectionsApi, EventNotifications, sprintf
     }
 
     function createFailure(response) {
-      EventNotifications.error(sprintf(__("There was an error creating catalog %s."), catalogObj.name));
+      EventNotifications.error(__(response.data.error.message));
```

Comment:
This might be a translation nightmare.  Wondering if we should do a message like:

There was an error creating catalog [NAME] <- translate this, then 
[translate] Server Responded: [/translate] [SERVER_ERROR_MESSAGE]

thoughts?